TwoPaneLayout/library/src/androidTest/java/com/microsoft/device/dualscreen/twopanelayout/TwoPaneTest.kt

```diff
@@ -137,6 +137,78 @@ class TwoPaneTest : LayoutTest() {
         var heightDp: Dp
         var hingeBoundsDp: DpRect
 
+        val drawLatch = CountDownLatch(2)
+        val childSize = arrayOfNulls<IntSize>(2)
+        val childPosition = arrayOfNulls<Offset>(2)
+        activityTestRule.setContent {
+            with(LocalDensity.current) {
+                widthDp = width.toDp()
+                heightDp = height.toDp()
+
+                val left = hingeBounds.left.toDp()
+                val top = hingeBounds.top.toDp()
+                val right = hingeBounds.right.toDp()
+                val bottom = hingeBounds.bottom.toDp()
+
+                hingeBoundsDp = DpRect(left, top, right, bottom)
+            }
+
+            Container(width = width, height = height) {
+                MockTwoPaneLayout(
+                    windowState = WindowState(
+                        hasFold = true,
+                        foldIsHorizontal = false,
+                        foldBoundsDp = hingeBoundsDp,
+                        foldIsSeparating = false,
```

Comment:
I think foldIsSeparating should be true for this test (since it uses the hinge bounds in the assertions below)

Reply:
or if we want it to test the case of a large screen/non-separating fold, the assertions and width/height should be changed to make sure it satisfies those conditions! I think 800 pixels is too small to be considered a large screen for us (and then I believe the assertions should use `width/2` instead of `hingeBounds.left`)